ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -257,65 +192,50 @@ Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by
 
 Now test the Connection to the server and type following:
 
-```
-pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY
-```
-If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **"CTRL+C"**.
+- **`# pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY`**
+
+If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **`"CTRL+C"`**.
 
 Now edit the environment file, open the file by typing:
 
-```
-nano /etc/environment
-```
+- **`# nano /etc/environment`**
 
 In the environment press down arrow in order to go to a new line and insert following line:
 
-```
-SERVER_ROOT=http://92.246.17.152:8080/uccs/inifile/
-```
+- **`# SERVER_ROOT=http://92.246.17.152:8080/uccs/inifile/`**
 
 Now press **"CTRL+O"** and press enter to save the file, afterwards exit it by pressing **"CTRL+X"**.
 
 Now run following commands, though NOTE: username and password are both testuser, also when typing password and username do not include **<>**.
 
-```
-export SERVER_ROOT
-```
+- **`# export SERVER_ROOT`**
 
 Type exit and press enter this will put you back to the very beginning, log back in as an Admin/Superuser/ROOT by typing following:
 
-```
-sudo su
-```
+- **`$ sudo -i`**
```

Comment:
Don't run the tests as root here. The thin-client-config-agent works under normal user privileges.

Furthermore, I switched over to our own script here. The thin-client-config-agent is +/- unmaintained by Canonical there days, so I forked it, too. See: 
https://github.com/ArcticaProject/remote-logon-config-agent

Please put remote-logon-config-agent here, instead of thin-client-config-agent.